interface-definitions/ntp.xml.in

```diff
@@ -43,12 +43,6 @@
                   <valueless/>
                 </properties>
               </leafNode>
-              <leafNode name="preempt">
```

Comment:
I have to admit I've never used this option. Is it unsupported in Chrony or simply not needed due to different approach?

Reply:
Unsupported by chrony